api/task.go

```diff
@@ -95,6 +101,27 @@ type TaskDatabaseSchemaUpdatePayload struct {
 	VCSPushEvent  *vcs.PushEvent   `json:"pushEvent,omitempty"`
 }
 
+// TaskDatabaseSchemaUpdateGhostSyncPayload is the task payload for gh-ost syncing ghost table.
+type TaskDatabaseSchemaUpdateGhostSyncPayload struct {
+	Statement     string         `json:"statement,omitempty"`
+	SchemaVersion string         `json:"schemaVersion,omitempty"`
```

Comment:
Likely we need migration type as well. Even for now we only support UI, we may later support VCS and provide via our CLI/Library

Reply:
The migration type should always be `MIGRATE`. Why would we need it?

Reply:
My bad. Misread